packages/core/networks/4.json

```diff
@@ -9,7 +9,7 @@
   },
   {
     "contractName": "IdentifierWhitelist",
-    "address": "0xCF744280D6ee7ca04e6D4167546a8463Ad899105"
+    "address": "0x7242892cc4653f1214cf700ddcc50ad47ebff564"
```

Comment:
THis is the ID whitelist hardcoded in the `MockOracleAncillary` that I use in tests, so I switched Finder back to it.